src/backend/distributed/connection/remote_commands.c

```diff
@@ -187,6 +187,35 @@ LogRemoteCommand(MultiConnection *connection, const char *command)
 
 /* wrappers around libpq functions, with command logging support */
 
+
+/*
+ * ExecuteCriticalRemoteCommand executes a remote command that is critical
+ * to the transaction. If the command fails then the transaction aborts.
+ */
```

Comment:
Hm - nothing in the content of this really seems to be related to "Critical"? Or is that just meant to refer to the ERROR level?